### DISCUSSION TITLE
Add scroll bar to beatmap listing

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#15921] [Ideas]
When being scrolled down a lot in the beatmap listing it is very tedious to get around the listings. Adding a scroll bar would be helpful, so you can quickly scroll to a certain point in the listing.

**Comment by frenzibyte** [comment]
I've removed your other suggestion from this discussion. Please open a separate discussion for that, it's not simple to manage discussions if they have multiple issues combined together.

**Comment by peppy** [comment]
I do think that most, if not every scroll view should have a scroll bar (even if it only shows after beginning to scroll via another means). The only cases it should be hidden completely are when there is no scroll content.
